### PR TITLE
[release-3.9] Persist oreg_url in node image

### PR DESCRIPTION
**Pull request by kargakis**
/hold

### CODE REVIEW

File: playbooks/azure/openshift-cluster/build_node_image.yml [hunk 1]
```diff
@@ -45,6 +45,12 @@
 - name: install openshift
   import_playbook: ../../openshift-node/private/image_prep.yml
 
+- name: persist oreg_url
+  copy:
+    dest: "/root/oreg_url"
```
Comment (jim-minter) [inline playbooks/azure/openshift-cluster/build_node_image.yml]: /etc ?

File: playbooks/azure/openshift-cluster/launch.yml [hunk 2]
```diff
@@ -19,8 +19,8 @@
       url: "{{ item }}"
       dest: "{{ tmp.path }}/"
     with_list:
-    - "http://acs-engine-build-acs-engine-build.svc.ci.openshift.org/acs-engine"
-    - "http://acs-engine-build-acs-engine-build.svc.ci.openshift.org/openshift.json"
+    - "http://acs-engine-build-acs-engine.svc.ci.openshift.org/acs-engine"
```
Comment (jim-minter) [inline playbooks/azure/openshift-cluster/launch.yml]: is this change meant to be here?

Comment (kargakis) [reply playbooks/azure/openshift-cluster/launch.yml]: Will be dropped once I see a successful test run.

https://ci.openshift.redhat.com/jenkins/view/All/job/test_pull_request_origin_extended_conformance_azure_39_test/1/console

**Comment by kargakis** [comment]
Got a clean run yesterday [here](https://ci.openshift.redhat.com/jenkins/view/All/job/test_pull_request_origin_extended_conformance_azure_39_test/6/consoleFull)

This should be ready to merge I think @kwoodson @jim-minter

**Comment by kargakis** [comment]
/assign pweil-

needs lgtm and approval

**Pull request merged**